manifests/4.4/local-volumes.crd.yaml

```diff
@@ -57,6 +57,21 @@ spec:
                   fsType:
                     description: File system type
                     type: string
+                  useAllDevices:
+                      description: Switch to enable selection of all available paths for storage
+                      type: boolean
+                  excludeDeviceNames:
+                    description: 'A list of regex patterns that match devices which would be excluded for local storage when useAllDevices is enabled.
+                    For example - ["/dev/sda", "/dev/sdb", "/dev/disk/by-id/ata-crucial"]'
```

Comment:
We could reject partitions too.

Reply:
As well as LVs.

Reply:
@leseb Based on a discussion yesterday, we are abandoning the idea of making this API change for now and exploring the possibility of another CRD for discovery in the future.

The reasoning was that changing the existing API with a new, more risky feature could be bad UX.